sdk/src/main/java/com/meniga/sdk/models/budget/MenigaBudget.java

```diff
@@ -185,4 +185,12 @@ public MenigaBudget createFromParcel(Parcel source) {
 	public static Result<MenigaBudget> create(BudgetType type, String name, String description, List<Long> accountIds, BudgetPeriod period, Integer periodOffset) {
 		return apiOperator.createBudget(type, name, description, accountIds, period, periodOffset);
 	}
+
+	public static Result<Void> delete(long budgetId) {
```

Comment:
Could you add instance methods as well for deleting and resetting? There should usually be an instance of the MenigaBudget for these operations.